### PR TITLE
Kubernetes enabled and user/groups advisory

### PR DESCRIPTION
**Pull request by stevenGravy**
Kubernetes agent and other installations often fail due to lack of k8s enabled and no user/groups in a user role.

### CODE REVIEW

File: docs/pages/includes/kubernetes-access/confirm-k8s.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+</Details>
+<Details
+title="Confirm User has Kubernetes access"
+scope={["cloud"]}
+scopeOnly={true}
+opened={true}
+>
+
+The Teleport user access the Kubernetes cluster will need Kubernetes groups or users
+to connect to the Kubernetes cluster as.
```
Comment (webvictim) [inline docs/pages/includes/kubernetes-access/confirm-k8s.mdx:45]:
```suggestion
The Teleport user accessing the Kubernetes cluster will need to be assigned Kubernetes groups
or users that they can use to make requests to the Kubernetes cluster.
```

File: docs/pages/includes/kubernetes-access/confirm-k8s.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<Details
+title="Confirm Kubernetes is enabled in Teleport"
+scope={["oss", "enterprise"]}
+scopeOnly={true}
+opened={true}
+>
+
+Confirm the `Kubernetes` status is `enabled` within the cluster and the user has Kubernetes groups
```
Comment (ptgott) [inline docs/pages/includes/kubernetes-access/confirm-k8s.mdx:8]: I think this could use a short introductory sentence/phrase for why we're confirming this information. 

For example, "Teleport Kubernetes Access must be enabled before you can use this guide. Confirm that..."

Comment (ptgott) [inline docs/pages/includes/kubernetes-access/confirm-k8s.mdx:8]:
```suggestion
Confirm that `tsh status` shows `Kubernetes:enabled` and the user has Kubernetes groups
```
To make this a bit more explicit

File: docs/pages/includes/kubernetes-access/confirm-k8s.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+If Kubernetes is `disabled` then the `kube_listen_addr` needs setting within the `proxy_service` as below. Restart
+the Teleport process after setting this. 
+
+```yaml
+proxy_service:
+  enabled: true
+  kube_listen_addr: 0.0.0.0:3026
+```
+
+See the [Teleport Kubernetes Access Controls](/docs/kubernetes-access/controls) for configuring Kubernetes
```
Comment (ptgott) [inline docs/pages/includes/kubernetes-access/confirm-k8s.mdx:33]:
```suggestion
See the [Teleport Kubernetes Access Controls](/docs/kubernetes-access/controls) guide for configuring Kubernetes
```

File: docs/pages/includes/kubernetes-access/confirm-k8s.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+RBAC to Teleport roles if the user has no `Kubernetes groups` or `Kubernetes users`.
+
+</Details>
+<Details
+title="Confirm User has Kubernetes access"
```
Comment (ptgott) [inline docs/pages/includes/kubernetes-access/confirm-k8s.mdx:38]:
```suggestion
title="Confirm that your user has Kubernetes access"
```
Small grammar tweak

File: docs/pages/includes/kubernetes-access/confirm-k8s.mdx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+scope={["cloud"]}
+scopeOnly={true}
+opened={true}
+>
+
+The Teleport user accessing the Kubernetes cluster will need to be assigned Kubernetes groups
```
Comment (ptgott) [inline docs/pages/includes/kubernetes-access/confirm-k8s.mdx:44]: Other text in this guide is addressed to "you" the reader. I think this paragraph should be as well.

File: docs/pages/includes/kubernetes-access/confirm-k8s.mdx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  Cluster:            example.teleport.sh
+  Roles:              access,editor
+  Logins:             bob
+  Kubernetes:         enabled
+  Kubernetes groups:  developer
+  Valid until:        2022-04-20 22:39:15 -0400 EDT [valid for 12h0m0s]
+  Extensions:         permit-X11-forwarding, permit-agent-forwarding, permit-port-forwarding, permit-pty
+```
+
+See the [Teleport Kubernetes Access Controls](/docs/kubernetes-access/controls) for configuring Kubernetes
```
Comment (ptgott) [inline docs/pages/includes/kubernetes-access/confirm-k8s.mdx:60]:
```suggestion
See the [Teleport Kubernetes Access Controls](/docs/kubernetes-access/controls) guide for configuring Kubernetes
```

File: docs/pages/includes/kubernetes-access/confirm-k8s.mdx [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+  Cluster:            teleport.example.com
+  Roles:              access,editor
+  Logins:             bob
+  Kubernetes:         enabled
+  Kubernetes groups:  developer
+  Valid until:        2022-04-20 22:39:15 -0400 EDT [valid for 12h0m0s]
+  Extensions:         permit-X11-forwarding, permit-agent-forwarding, permit-port-forwarding, permit-pty
+```
+
+If Kubernetes is `disabled` then the `kube_listen_addr` needs setting within the `proxy_service` as below. Restart
```
Comment (zmb3) [inline docs/pages/includes/kubernetes-access/confirm-k8s.mdx:24]: "needs setting" sounds awkward here. I would suggest rewording this.

**Comment by stevenGravy** [comment]
Addressed in https://github.com/gravitational/teleport/pull/12682
